core/modules/system/tests/system.test

```diff
@@ -2497,7 +2503,6 @@ class SystemValidTokenTest extends BackdropWebTestCase {
     // The following checks will throw PHP notices, so we disable error
     // assertions.
     $this->assertErrors = FALSE;
-    $this->assertFalse(backdrop_valid_token(NULL, new stdClass()), 'Token NULL, value object returns FALSE.');
```

Comment:
This edge-case I couldn't find a fix for. The second parameter of `backdrop_valid_token()` expects a string. Casting an object to a string now throws a notice, but as this situation should not occur (and PHP now throws the notice if it does occur), removing the check seemed appropriate.